### DISCUSSION TITLE
How to convert SVGZ to JPG?

### DISCUSSION THREAD

**Discussion by TheElectroclassics** [ImageMagick/ImageMagick#3199] [Help]
I have a folder full of SVGZ images that I want to rasterize to JPG. I tried running `magick *.svgz convert.jpg` inside the folder but of course, it didn't work. So, how can I raster 134 SVGZ files to JPG in one sitting?

**Comment by urban-warrior** [comment]
Try
```
magick mogrify -format jpg *.svgz
```

**Reply by TheElectroclassics** [reply]
It didn't work. It only appends "jpg" to the file name.

**Comment by urban-warrior** [comment]
That would be a bug.  Its producing a gzipped jpeg file.  The problem appears to be localized to SVGZ.  We tried TIFF to JPEG and the command worked fine. We'll add a patch to fix this issue within a day or two.  In the mean-time, use a script.  For tcsh, for example:

```
foreach file (*.svgz)
  magick $file $file:t.jpg
  echo $file:t.jpg
end
```
